R/bind.r

```diff
@@ -150,8 +91,33 @@ bind_rows <- function(..., .id = NULL) {
   out
 }
 
+#' Bind multiple data frames by column
+#'
+#' @description
+#' Bind any number of data frames by column, making a wider result.
+#' This is similar to `do.call(cbind, dfs)`.
+#'
+#' Where possible prefer using a [join][left_join] to combine multiple
+#' data frames. `bind_cols()` binds the rows in order in which they appear
+#' so it is easy to create meaningless results without realising it.
+#'
+#' @inheritParams bind_rows
+#' @param ... Data frames to combine. Each argument can either be a data frame,
+#'   a list that could be a data frame, or a list of data frames.
+#'   Inputs are [recycled][vctrs::vector_recycling_rules] to the same length,
+#'   then matched by position.
+#' @inherit bind_rows return
+#' @param .name_repair One of `"unique"`, `"universal"`, or
+#'   `"check_unique"`. See [vctrs::vec_as_names()] for the meaning of these
+#'   options.
```

Comment:
```suggestion
#' @param .name_repair One of `"unique"`, `"universal"`, or
#'   `"check_unique"`. See [vctrs::vec_as_names()] for the meaning of these
#'   options.
#' @inherit bind_rows return
```

To keep params together

Reply:
Done by hand